.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please pull the latest master, I removed `.idea` folder completely after your last PR with folding. The project already contains `.editorconfig` to specify formatting settings so maybe it's better not to have both.